src/NBomber/DomainServices/Reporting/HtmlReport.fs

```diff
@@ -33,32 +33,32 @@ module AssetsUtils =
 
 let private applyHtmlReplace (nBomberInfoJsonData: string) (testInfoJsonData: string)
                              (statsJsonData: string) (timeLineStatsJsonData: string)
-                             (line: string) =
+                             (hintsJsonData: string) (line: string) =
     let removeLineCommand = "<!-- remove-->"
     let includeViewModelCommand = "<!-- include view model -->"
     let includeAssetCommand = "<!-- include asset -->"
 
     if line.Contains(removeLineCommand) then
         String.Empty
     else if line.Contains(includeViewModelCommand) then
-        sprintf "const viewModel = { nBomberInfo: %s, testInfo: %s, statsData: %s, timeLineStatsData: %s };"
-                nBomberInfoJsonData testInfoJsonData statsJsonData timeLineStatsJsonData
+        sprintf "const viewModel = { nBomberInfo: %s, testInfo: %s, statsData: %s, timeLineStatsData: %s, hints: %s };"
```

Comment:
please use F# 5 string interpolation for newly created strings or when you update an old one